addons/clones/spriteCount.js

```diff
@@ -0,0 +1,67 @@
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+  const ogCloneCounter = vm.runtime.changeCloneCounter;
+
+  async function updateCounts() {
+    let counts = {};
+    let remove = addon.self.disabled || !addon.settings.get("showSpriteCount");
+
+    vm.runtime.targets
+      .filter((target) => !target.isOriginal)
+      .map((target) => target.sprite.name)
+      .forEach((target) => {
+        if (!counts[target]) counts[target] = 1;
+        else counts[target] += 1;
+      });
+
+    const spriteWrapper = await addon.tab.waitForElement("[class*=sprite-selector_items-wrapper]");
+    const spriteNames = Array.from(spriteWrapper.querySelectorAll("[class*=sprite-selector-item_sprite-name]"));
```

Comment:
we shouldnt* need a waitForElement here. You can probably merge these 2 lines into one and just use one querySelectorAll

Reply:
We do, i dont know why but we do need the wait for element

Reply:
Or we dont? wtf